components/chassis.py

```diff
@@ -251,9 +250,7 @@ def __init__(
         self, track_width: float, wheel_base: float, swerve_config: SwerveConfig
     ) -> None:
         self.imu = Pigeon2(0)
-        self.heading_controller = ProfiledPIDControllerRadians(
-            3, 0, 0, TrapezoidProfileRadians.Constraints(100, 100)
-        )
+        self.heading_controller = PIDController(3.0, 0.0, 0.0)
```

Comment:
Do you think we should use the same PID gains (and hence controller) for auto too?

Reply:
That is where this originally started - this used to be the controller we used in auto.
We could use the same one.
@LucienMorey your thoughts?

Reply:
That suggestion is fine. The only reason we have diverged is because we had a trapezoidal one in the component and needed pure pid in auto.